### PR TITLE
Add debug option

### PR DESCRIPTION
**Pull request by jesusbv**
## Description

Adding a debug flag/option in order to
be able to skip some verifications when
using rmt-manual-instance-verify when
debugging certain issues

This Fixes bsc#1210160

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [X] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [X] I have verified that my code follows RMT's coding standards with `rubocop`.
- [X] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [X] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/322613/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 454537ec0e2225bcaf288f3c0a17de9147cf67b5 into fe2c583173c896d6c522c3b18fbb6b802e9559b9)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/323281/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests.
Failed fast in project rmt on step docker-compose-run-tests. (Merged 454537ec0e2225bcaf288f3c0a17de9147cf67b5 into e88e8d75cb85968c1b42983fc4595afab5b64850)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/323393/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 454537ec0e2225bcaf288f3c0a17de9147cf67b5 into 08e9887e0deb07d897f10819548b40a93d5b902e)

**Review by rjschwei — APPROVED** [review]

File: bin/rmt-manual-instance-verify [hunk 1]
```diff
@@ -110,7 +110,7 @@ raise 'Instance verification provider is missing or failed to initialize' unless
 puts "Loaded instance verification module: #{InstanceVerification.provider}"
 
 # Verifying instance data
-provider = InstanceVerification.provider.new(logger, request, product_hash, instance_data)
```
Comment (digitaltom) [inline bin/rmt-manual-instance-verify:113]: This fails in lint, please use a keyword argument for boolean parameters: https://rubystyle.guide#boolean-keyword-arguments

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/324505/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 5f68b0e11d05e8b0138daaf22bcda8a13a5a08ff into 08e9887e0deb07d897f10819548b40a93d5b902e)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/328287/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 7c8b29d52ef83378ede2cdb2099fdebbb85935e7 into eea5e93c4380bed7bd0d06e7ef8439dd8d022118)

**Review by digitaltom — APPROVED** [review]

**Pull request merged**